alecs-theme/src/WEB-INF/src/resources-importer/journal/templates/Contact Us/Contact Us.ftl

```diff
@@ -0,0 +1,45 @@
+<style>
+    .portlet-title-text{
+        display: none;
+    }
+    .row {
+        margin-top: 1rem;
+    }
+    .header-img {
+        border-top-left-radius: 15px;
+        border-top-right-radius: 15px;
+    }
+    form {
+        width: 100%;
+        display: contents;
+    }
+    textarea {
+        min-height: 50px;
+    }
+</style>
+<#if BackgroundImage.getData()?? && BackgroundImage.getData() != "">
+    <img alt="${BackgroundImage.getAttribute("alt")}" data-fileentryid="${BackgroundImage.getAttribute("fileEntryId")}" src="${BackgroundImage.getData()}" class="header-img"/>
+</#if>
+<div class="row">
+  <form action="${FormBackend.getData()}">
+    <p class="field col-sm-6">
+      <label for="first">First Name: </label>
+      <input type="text" id="first" name="first_name" class="form-control" placeholder="Life"/>
+    </p>
+    <p class="field col-sm-6">
+      <label for="last">Last Name: </label>
+      <input type="text" id="last" name="last_name" class="form-control" placeholder="Ray"/>
```

Comment:
We should not hardcode a placeholder value unless it is customizable since this could be wrong in other languages.  Allow the content administrator to manage this if possible.